### ISSUE TITLE
Reader: images from Gutenberg are not sized correctly

### ISSUE THREAD

**Issue by koke** [wordpress-mobile/WordPress-iOS#10867]
Moved from https://github.com/wordpress-mobile/gutenberg-mobile/issues/467

When creating a post in gutenberg-mobile that has some images, the reader tries to resize them in such a way that it fits the viewport. Currently, it seems that the mobile block editor doesn't provide the proper markup to let the wpios Reader succeed in the resizing.

This is only a wpios issue as the wpandroid Reader seems to be able to resize the images.

See these posts in the Reader to verify:

Post written on wpios: https://stefanosuser.wordpress.com/2019/01/07/two-images/
Post written on the web: https://stefanosuser.wordpress.com/2019/01/07/two-images-web/
Post written on wpandroid: https://stefanosuser.wordpress.com/2019/01/07/two-images-android/

### Expected behavior

![screen shot 2019-01-21 at 13 11 38](https://user-images.githubusercontent.com/8739/51473966-1c561880-1d7e-11e9-8a98-115d5b4f2855.png)

### Actual behavior

![screen shot 2019-01-21 at 13 11 29](https://user-images.githubusercontent.com/8739/51473972-211acc80-1d7e-11e9-9ac8-c52fa633a581.png)

**Comment by koke** [comment]
I debugged this a bit and noticed that the post created by gutenberg-mobile doesn't include either the `size` or `data-orig-size` attributes. Those are also missing in Gutenberg on a self hosted site, but somehow get added by Jetpack in the frontend (photon?).

In any case, I think the reader should default to 100% width if the size of an image is unknown.

**Comment by aerych** [comment]
> In any case, I think the reader should default to 100% width if the size of an image is unknown.

I'd suggest the image's native width (adjusted for retina displays) or 100% readable content margin width, which ever is smaller in order to avoid pixelation from upscaling.

**Comment by designsimply** [comment]
Still valid. I agree with @koke and @aerych that images in the Reader should default to a max-width of 100% width if the size of an image is unknown.

I tested and was unable to reproduce the problem using the following testing steps:

1. Go to My Site > Blog Posts > +.
1. Add a title, content, and an image.
1. Publish the post.
1. Find the post in the Reader.

```
<!-- wp:image {"id":747,"sizeSlug":"large"} -->
<figure class="wp-block-image size-large"><img class="wp-image-747" src="https://madefortesting190318.files.wordpress.com/2020/03/pexels-photo-1161547.jpg?w=1024" alt="" /></figure>
<!-- /wp:image -->
```

<img src="https://user-images.githubusercontent.com/1119271/76906565-d3a6cc00-6869-11ea-9711-9c8e754facd0.PNG" width="270" alt="IMG_3558"> <img src="https://user-images.githubusercontent.com/1119271/76906574-dacdda00-6869-11ea-986d-b58d80cabb3f.PNG" width="270" alt="IMG_3560">
<sup>Tested with WPiOS 14.4.0.0 TestFlight beta on iPhone 6S iOS 13.3.1.</sup>

**Result:** my image looked fine in the Reader, likely because it has whatever required formatting that the Reader currently uses to set a max-width for images.

However, the issue where the image expands outside the edges of the Reader view was reported in Slack by monicavohara today and the image html (url and caption have been changed) looked like this:

```
<!-- wp:image -->
<figure class="wp-block-image"><img src="image_url?fm=png&auto=format" alt="Image result for m&A" /><figcaption>caption</figcaption></figure>
<!-- /wp:image -->
```

**Conclusion:** we need to come up with a better way to enforce the max-width for images in the Reader regardless of where they were originally created.

<sup>h/t @elibud and monicavohara for the report.</sup>
